Airports/EGSC/Sector_Ownership.txt

```diff
@@ -1,36 +1,36 @@
 SECTOR:Only SCTWR 1:0:0
-OWNER:ESS:SCR:SCA:SCT
+OWNER:ESS:SCA:SCT
 
 SECTOR:Only SCTWR 2:0:0
-OWNER:ESS:SCR:SCA:TCNE:TCNW:LOREL:TCN:TCM:TCSW:TCSE:TCS:TCE:SABER:DAGGA:REDFA:JACKO:TC:LML:LME:LM:LC:LES:LEN:LE:LDL:LD:LSF:LS:LWN:LSC:L:LMEN:O
+OWNER:ESS:SCA:TCNE:TCNW:LOREL:TCN:TCM:TCSW:TCSE:TCS:TCE:SABER:DAGGA:REDFA:JACKO:TC:LML:LME:LM:LC:LES:LEN:LE:LDL:LD:LSF:LS:LWN:LSC:L:LMEN:O
 
 SECTOR:Only SCAPP:0:0
-OWNER:SCR:SCA
+OWNER:SCA
 
 SECTOR:SCTWR:0:2000
-OWNER:SCT:SCR:SCA:SSR:ESS:LOREL:TCNE:TCN:TC:DAGGA:REDFA:TCE:LEN:LE:LC:LSC:L
-ALTOWNER:Observing LTC_N_CTR:SCT:SCR:SCA:SSR:ESS:TCNE:O:LMEN:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
-ALTOWNER:Observing LTC_CTR:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:O:LMEN:TC:SABER:TCE:LEN:LE:LC:LSC:L
-ALTOWNER:Observing LTC_E_CTR:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:O:LMEN:TCE:LEN:LE:LC:LSC:L
-ALTOWNER:Observing LON_E_CTR:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:O:LMEN:LE:LC:LSC:L
-ALTOWNER:Observing LON_C_CTR:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:O:LMEN:LC:LSC:L
-ALTOWNER:Observing LON_SC_CTR:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:O:LMEN:LSC:L
-ALTOWNER:Observing London FIR:O:LMEN:SCT:SCR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
-ALTOWNER:Mentoring TWR/APP:SCTM:SCT:SCRM:SCR:SCAM:SCA:SSRM:SSR:ESSM:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
+OWNER:SCT:SCA:SSR:ESS:LOREL:TCNE:TCN:TC:DAGGA:REDFA:TCE:LEN:LE:LC:LSC:L
+ALTOWNER:Observing LTC_N_CTR:SCT:SCA:SSR:ESS:TCNE:O:LMEN:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
+ALTOWNER:Observing LTC_CTR:SCT:SCA:SSR:ESS:TCNE:TCN:O:LMEN:TC:SABER:TCE:LEN:LE:LC:LSC:L
+ALTOWNER:Observing LTC_E_CTR:SCA:SSR:ESS:TCNE:TCN:TC:SABER:O:LMEN:TCE:LEN:LE:LC:LSC:L
+ALTOWNER:Observing LON_E_CTR:SCT:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:O:LMEN:LE:LC:LSC:L
+ALTOWNER:Observing LON_C_CTR:SCT:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:O:LMEN:LC:LSC:L
+ALTOWNER:Observing LON_SC_CTR:SCT:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:O:LMEN:LSC:L
+ALTOWNER:Observing London FIR:O:LMEN:SCT:SCA:SSR:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
+ALTOWNER:Mentoring TWR/APP:SCTM:SCAM:SCA:SSRM:SSR:ESSM:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
```

Comment:
```suggestion
ALTOWNER:Mentoring TWR/APP:SCTM:SCT:SCAM:SCA:SSRM:SSR:ESSM:ESS:TCNE:TCN:TC:SABER:TCE:LEN:LE:LC:LSC:L
```
SCT should be retained here.

If you're removing the `SCRM` from here (which I think is the right thing to do), I don't see why we wouldn't also remove that position definition from https://github.com/VATSIM-UK/UK-Sector-File/blob/main/Airports/EGSC/Positions_Mentor.txt